### PR TITLE
Traefik routing on Alkemio Server

### PR DESCRIPTION
**Pull request by valentinyanakiev**
- Updated alkemio.yml to run with default values (no .env file) with quickstart-services.yml composed. Default configuration assumes everything is exposed through traefik.
- Updated README.MD with a fix for failing synapse configuration locally.
- Removed hack with explicit docker network. Replaced with explicit mapping of the docker host network and the host gateway inside the containers that need it (needs to be tested on Win).
- Added routing for the client so that the client can be started out-of-the-box with running server configuration, re-using traefik routing

**P.S. Only approve, do not merge as infra-operations needs an update.**

### CODE REVIEW

File: .build/ory/kratos/email-password/kratos.yml [hunk 1]
```diff
@@ -4,70 +4,53 @@ dsn: memory
...
 
 selfservice:
   default_browser_return_url: http://localhost:3000/
   whitelisted_return_urls:
     - http://localhost:3000
 
   methods:
-    link:
-      enabled: true
-
```
Comment (atstoyanov) [inline .build/ory/kratos/email-password/kratos.yml]: Used for sending recovery and verification emails.

File: .build/ory/kratos/email-password/kratos.yml [hunk 2]
```diff
@@ -4,70 +4,53 @@ dsn: memory
...
-      ui_url: http://localhost:3000/auth/recovery
+      ui_url: http://localhost:3000/identity/recovery
 
     verification:
       enabled: true
-      ui_url: http://localhost:3000/auth/verify
+      ui_url: http://localhost:3000/identity/verify
       after:
-        default_browser_return_url: http://localhost:3000/auth/verify/success
+        default_browser_return_url: http://localhost:3000/
```
Comment (atstoyanov) [inline .build/ory/kratos/email-password/kratos.yml]: /identity/verify/success

File: .build/ory/kratos/email-password/kratos.yml [hunk 3]
```diff
@@ -4,70 +4,53 @@ dsn: memory
...
       after:
-        default_browser_return_url: http://localhost:3000/
+        default_browser_return_url: http://localhost:3000/identity/login
 
     login:
-      ui_url: http://localhost:3000/auth/login
+      ui_url: http://localhost:3000/identity/login
       lifespan: 10m
-      after:
-        default_browser_return_url: http://localhost:3000/auth/login/success
```
Comment (atstoyanov) [inline .build/ory/kratos/email-password/kratos.yml]: Should stay:
http://localhost:3000/auth/identity/success

File: .build/ory/kratos/email-password/kratos.yml [hunk 4]
```diff
@@ -4,70 +4,53 @@ dsn: memory
...
 
     registration:
       lifespan: 10m
-      ui_url: http://localhost:3000/auth/registration
+      ui_url: http://localhost:3000/identity/registration
       after:
-        default_browser_return_url: http://localhost:3000/auth/registration/success
```
Comment (atstoyanov) [inline .build/ory/kratos/email-password/kratos.yml:70]: Should stay

File: quickstart-services.yml [hunk 5]
```diff
@@ -51,12 +64,13 @@ services:
...
     ports:
-      - "4433:4433" # public
-      - "4434:4434" # admin
+      - 4433
+      - 4434
     restart: unless-stopped
     environment:
       - DSN=mysql://root:${MYSQL_ROOT_PASSWORD}@tcp(${DATABASE_HOST}:3306)/kratos?max_conns=20&max_idle_conns=4
       - LOG_LEVEL=trace
+      - SERVE_PUBLIC_BASE_URL=http://localhost:3000/identity/ory/kratos/public
```
Comment (atstoyanov) [inline quickstart-services.yml]: Already configured from the kratos.yml

**Review by atstoyanov — CHANGES_REQUESTED** [review]

File: .build/ory/kratos/email-password/kratos.yml (hunk 1 above)
Comment (valentinyanakiev) [reply .build/ory/kratos/email-password/kratos.yml]: Addressed in ff3914cf670151f476fbe9bdf9b747d578aec552

File: .build/ory/kratos/email-password/kratos.yml (hunk 2 above)
Comment (valentinyanakiev) [reply .build/ory/kratos/email-password/kratos.yml]: Addressed in ff3914cf670151f476fbe9bdf9b747d578aec552

File: .build/ory/kratos/email-password/kratos.yml (hunk 3 above)
Comment (valentinyanakiev) [reply .build/ory/kratos/email-password/kratos.yml]: Addressed in ff3914cf670151f476fbe9bdf9b747d578aec552

File: .build/ory/kratos/email-password/kratos.yml (hunk 4 above)
Comment (valentinyanakiev) [reply .build/ory/kratos/email-password/kratos.yml:70]: Addressed in ff3914cf670151f476fbe9bdf9b747d578aec552. Replaced **auth** with **identity**

File: quickstart-services.yml (hunk 5 above)
Comment (valentinyanakiev) [reply quickstart-services.yml]: Addressed in ff3914cf670151f476fbe9bdf9b747d578aec552. Removed.

**Review by atstoyanov — APPROVED** [review]

**Pull request merged**